### DISCUSSION TITLE
Soliciting Ideas for Rebranding VA-Spec

### DISCUSSION THREAD

**Discussion by larrybabb** [ga4gh/va-spec#236] [Ideas]
There have been discussions over the past year or 3 about whether VA-Spec was the appropriate name for this product.  VA-Spec is a core information model that support representation of evidence-based statements for computationally sharing scientific results, methods and the assertions that can be drawn from applying those methods. From the outset we have always presumed that the subject of those results and assertions would be "variants".  The term Annotation may be useful in generalizing the idea of results, assertion and/or knowledge about various genomic subjects like variants, but it might be to basic to accurately represent the core im for knowledge and results around genetic concepts like variants.

So, before we go much further in formalizing this product we should determine if a better brand or name should be applied.

This discussion is looking for ideas on what that name or brand should be.

Please offer arguments in either direction for changing the name and/or provide proposals for a new brand name.

**Comment by korikuzma** [comment]
@larrybabb I gave ChatGPT a prompt to come up with names given "core information model that support representation of evidence-based statements for computationally sharing scientific results, methods and the assertions that can be drawn from applying those methods".

**Acronyms Emphasizing Core Attributes**

1. CAMEO (Core Assertion Model for Evidence and Observations)
2. FRAME (Framework for Representing Assertions and Methods of Evidence)
3. ASSERT (Assertion Sharing for Scientific Evidence, Results, and Truths)
4. ECHO (Evidence, Computation, and Hierarchical Observations)
5. TRACE (Transparency in Representation of Assertions and Computational Evidence)

**Playful Acronyms**

1. PINE (Provenance, Information, and Evidence)
2. BEAR (Building Evidence and Assertions Reliably)
3. LENS (Logical Evidence and Assertion Sharing)
4. LEAF (Logical Evidence and Assertions Framework)
5. SPROUT (Scientific Provenance and Representation Of Useful Truths)

I like BEAR simply because we could have a really cute logo.

**Reply by korikuzma** [reply]
Other acronyms for BEAR:

1. Basis for Evidence and Assertion Rules
2. Building Evidence and Assertions Repository
3. Building Evidence and Assertion Records

Tagging @mbrush for thoughts

**Reply by korikuzma** [reply]
Annotating and Building Evidence and Assertions Reliably (AnnoBEAR)

**Comment by brendanreardon** [comment]
What about simply "GKS Schema" or "GKS Annotation Schema"? 

> From the outset we have always presumed that the subject of those results and assertions would be "variants". The term Annotation may be useful in generalizing the idea of results, assertion and/or knowledge about various genomic subjects like variants, but it might be to basic to accurately represent the core im for knowledge and results around genetic concepts like variants.

I agree that there is no reason that the schema, as is, cannot be applied to other classes of biomarkers that aren't "variants" or "genomic"; e.g., immune marker status. Leaving "variant" out of the branding may be helpful... although the "genomic" in GKS and GA4GH implies a scope 😵‍💫 This isn't too much of an issue though, I think.

**Comment by mbrush** [comment]
I think that whether (and to what) we change the name of the VA-Spec depends on how we see the scope, growth, and partitioning of specification content playing out in the future, if and when a spec for a broader modeling framework is released. 
 
This is how I see things. What we are releasing now in v1.0  is simply a specification for how to represent specific types of knowledge about genetic variation.  Yes, there is a more generic model and profiling methodology behind this, and we do envision a future release of this as a framework in which this more general model and methodology could be applied in other domains - but this has yet to be realized, and it is not clear yet how this framework might be implemented and organized, and how broad its scope and use will be.  
 
Importantly, I think that whatever this broader framework may end up looking like, we would still want to carve out a set of schema generated using the framework that provides a specification for representing variation knowledge - which we would want to continue to call "VA-Spec".   Since it is this narrowly scoped specification that we are submitting for v1 in April, it makes sense IMO to keep the name "VA-Spec" for this release.
 
If/when we release the broader framework for GA4GH, it will be provided as a separate/new specification. This new specification would include the generic core model content (currently in va-core-source), and the existing 'VA-Spec' would have to be refactored to import this core content from wherever it ends up as part of this new product - so it can be used/extended in VA profiles. But again, the set of VA Profiles will persist as a domain specific application of this larger framework - which we could continue to call the 'VA-Spec'.
 
Notably, I think that the name "VA-Spec'" could accommodate inclusion of Statement / Study Result profiles that are about things that are not Variants (e.g. Genes, Diseases, Treatments) - as long as these Statements / Results represent kinds of information that are considered as evidence for generating knowledge about genetic variation. So even with the name "VA-Spec", the specification can include profiles that are about non-variation entities/concepts. 
 
-----
 
That said, while it is not my preferred option I would not be against broadening the stated scope of the 1.0 specification explicitly from the outset, in anticipation of its being applied toward representation of genomic knowledge / information about things unrelated to variation. In this case, I would follow the recommendation of @brendanreardon above - which might give us a name like:  
- 'Genomic Knowledge Annotation Specification'  (GKA-Spec, or GKAS)
- 'Genomic Knowledge Standards Annotation Specification'  (GKS-AS)
- 'Genomic Knowledge Annotation Specification'  (GKAS)
 
In this scenario, as above, the 'GKA-Spec' includes the core model and the set of profiles created using the framework. But if/when such a framework is ready for release, the framework and core model become part of a separate 'modeling framework' product, and the  'GKA-Spec' would be considered an application of the framework to produce models for a particular domain, that includes just the domain-specific profiles.

**Reply by ahwagner** [reply]
I think that "Annotation Specification" is too broad. I would be in favor of something that places the focus on genomic evidence and classification though. Perhaps the _Genomic Evidence Modeling Specification (GEMS)_?

**Reply by korikuzma** [reply]
That's coool @ahwagner
